src/test/acceptance/README.md

```diff
@@ -28,10 +28,12 @@ describe('', () => {
       $('body').html().should.containEql('These Terms of Use')
     })
   })
-````
+```
 
 ## Fixtures
 
+_**Note:** The description below appears to be obsolete. See [this thread](https://artsy.slack.com/archives/C2TQ4PT8R/p1570211335035500) in the meantime for a workaround for capturing fixtures._
```

Comment:
Could probably be more aggressive in cutting stale docs here — I just was not really familiar with the state of this `acceptance-record` code (other than that it didn't work for me 😛 )